tools/cloud-build/daily-tests/tests/slurm-v5-hpc-centos7.yml

```diff
@@ -20,6 +20,8 @@ deployment_name: "cent-v5-{{ build }}"
 # non-alphanumeric chars and is capped at 10 chars.
 slurm_cluster_name: "centv5{{ build[0:4] }}"
 zone: us-central1-a
+cli_deployment_vars:
+  enable_cleanup_compute: true
```

Comment:
are there no other slurm tests, why just adding to this one?

Reply:
I don't see any other reference of hpc-slurm.yaml BP https://github.com/search?q=repo%3AGoogleCloudPlatform%2Fhpc-toolkit+path%3A%2F%5Etools%5C%2Fcloud-build%5C%2Fdaily-tests%5C%2Ftests%5C%2F%2F++%22hpc-slurm.yaml%22&type=code in other tests.

Reply:
I would look for all tests using [slurm-integration-test.yml](https://github.com/GoogleCloudPlatform/hpc-toolkit/pull/1767/files#diff-7cfee9141f93a04274d3ba1b990b0bf685d9481fdc364c8d90d96767491a376c)